README.md

```diff
@@ -4,9 +4,21 @@
 and systems administration tasks. It allows the user to run commands across a
 server farm.
 
-## Local usage
+## Local usage and environment setup
+
+The tool is intended to be configured and run on your local workstation/laptop. You should use `virtualenv` to create an isolated Python environment.
```

Comment:
While I don't necessarily agree with your reasoning, I'm happy for this to be merged if that is added.